src/main/java/com/jcabi/github/mock/MkIssueLabels.java

```diff
@@ -182,4 +217,15 @@ private String xpath() {
         );
     }
 
+    /**
+     * Returns a set of all of the issue's labels.
+     * @return Set of label names
+     */
+    private Set<String> labelSet() {
```

Comment:
@cvrebert As far as I can tell, there's no particular reason why it should declare the return type `Set`, let's make the it `Collection` instead. And let's name this method `labels()`, see [our method naming convention here](http://www.yegor256.com/2014/04/27/typical-mistakes-in-java-code.html#method-names).